### DISCUSSION TITLE
Mouse click based on Mac OS UI element provided

### DISCUSSION THREAD

**Discussion by tejaskumark** [go-vgo/robotgo#576] [General]
Hi Experts,

I want to click on Mac OS notification based on Mac OS UI element provided for e.g  in context of Mac OS

application (NSApplication) - XYZ
dialogue (NSWindow) - ABC
group NSBox - Empty description
button NSButtonCell - Continue
button NSButtonCell - Exit

**Comment by breece1991** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。
